Application/EdFi.Common/Security/Pbkdf2HmacSha1SecureHasher.cs

```diff
@@ -47,24 +54,13 @@ public PackedHash ComputeHash(string secret, int hashAlgorithm, int iterations,
             }
 
             return new PackedHash
-                   {
-                       Format = 0, HashAlgorithm = hashAlgorithm, HashBytes = bytes, Iterations = iterations, Salt = salt
-                   };
-        }
-
-        protected override bool CanHandleRequest(SecureHashRequest request)
-        {
-            // to allow for auto upgrading of an existing hash routine that used the string version of GetHashCode()
-            // we need to keep it in this check. Note, the string version of GetHashCode() is not deterministic when outside of the app domain.
-            return request.HashAlgorithm == HashHelper.GetSha256Hash(Algorithm).ToInt32()
-                   || request.HashAlgorithm == Algorithm.GetHashCode();
-        }
-
-        protected override PackedHash HandleRequest(SecureHashRequest request)
-        {
-            return request.Salt != null
```

Comment:
yes we use the default settings object for setting ujp the salt etc it is injected into the container.

Reply:
It looks like we never actually call this with a null Salt because the only caller gets the Salt from an actual hash after its unpacked.  Note if we ever did call this method with a null Salt, the Rfc2898DeriveBytes constructor would throw a null argument exception, so this is probably safe enough.

Reply:
however, i thought about this more last night, and i think i will add this check in the secret verifier

Reply:
done